test/CommandTest.php

```diff
@@ -98,39 +98,66 @@ public function testCanvasWidthAndHeightNullByDefault()
         $this->assertNull($command->canvasHeight());
     }
 
-    public function testFullTrimMod()
-    {
-        $command = self::command('trimAABBCCx2');
-        $this->assertEquals('AABBCC', $command->trimColor());
-        $this->assertEquals('2', $command->trimFuzz());
+    /**
+     * @param string $strCommand
+     * @param string $color
+     * @param string $fuzz
+     * @dataProvider dpTestOnlyTrimColor
+     */
+    public function testTrim($strCommand, $color, $fuzz)
+    {
+        $command = self::command($strCommand);
+        $this->assertEquals($color, $command->trimColor());
+        $this->assertEquals($fuzz, $command->trimFuzz());
+        $this->assertEquals($strCommand, (string) $command);
+    }
+
+    public function dpTestOnlyTrimColor()
+    {
+        return [
+            [
+                'trimAABBCC',
+                'AABBCC',
+                ''
+            ],
+            [
+                'trimFFF',
+                'FFF',
+                ''
+            ],
+            [
+                'trimAABBCCx2',
+                'AABBCC',
+                '2'
+            ],
+            [
+                'trimFFFx50',
+                'FFF',
+                '50'
+            ],
+            [
+                'trimx10',
+                '',
+                '10'
+            ],
+            [
+                'trim',
+                '',
+                ''
+            ]
+        ];
     }
 
-    public function testOnlyTrimColor()
-    {
-        $command = self::command('trimAABBCCx');
```

Comment:
Please return test case for this `'trimAABBCCx'` command.

Reply:
done